### PR TITLE
added docs for Brocade firmware 08.0.80 and above

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Added Brocade configuration guide for firmware 08.0.80 and above

# Impacts
Documentation

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -901,8 +901,68 @@ Where `xx` is the switch port number
...
+    dot1x enable
+    dot1x enable ethe 1/1/1
+    dot1x port-control auto ethe 1/1/1
+    dot1x macauth-override
+    dot1x timeout tx-period 3
+    dot1x timeout quiet-period 2
+    mac-authentication enable
+    mac-authentication enable ethe 1/1/1
+
+The configuration above enables authentication on port 1/1/1, make sure you change this to the ports where you want to perform enforcement
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: missing trailing dot?
I would also write:

"... enables authentication on port 1/1/1 - make sure..."

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 2]
```diff
@@ -901,8 +901,68 @@ Where `xx` is the switch port number
...
+ * Definition, Type: Brocade Switches
+ * RADIUS, Secret Passphrase: useStrongerSecret
+ * SNMP, Version: v2c
+ * SNMP, Community Read: public
+ * SNMP, Community Write: private
+
+VoIP
+++++
+
+In order to enable VoIP, you first need to enable LLDP then defined the network policy for tagging VoIP traffic on the ports where PacketFence is enabled.
```
Comment (extrafu) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: then define

**Comment by julsemaan** [comment]
I've made the adjustments, should we merge the docs or postpone to 8.3 ?

**Comment by extrafu** [comment]
You've got your answer :)

**Pull request merged**